main/kotlin/com/example/people_managment_system/repositories/CompanyRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.people_managment_system.repositories

```

Comment:
Seems like you have four domains in this service: Company, Employee, Manager and Team. 

There many ways to structure the project and it's up to you on how you want to do it. A recommendation is that you have four packages in `domain`. Something like: `domain.company`, `domain,employee`, `domain.manager` and `domain.team`. 

You can also structure it under an `.../infrastructure` package, this one is in the name basically logic or code that executes such as writing to a Database.